lib/store/src/csf/prepareStory.ts

```diff
@@ -194,7 +194,8 @@ export function prepareStory<TFramework extends AnyFramework>(
 
     return decoratedStoryFn(finalContext);
   };
-  const playFunction = storyAnnotations.play;
+  const playFunction =
+    storyAnnotations.play || componentAnnotations.play || projectAnnotations.play;
```

Comment:
The play function is not included in the project level, can you remove it?
```suggestion
    storyAnnotations.play || componentAnnotations.play;
```